### ISSUE TITLE
PdfTextExtractor extracts blank text

### ISSUE THREAD

**Issue by spstrademark** [syncfusion/flutter-widgets#1444]
Hello,
i am using the following versions in flutter and syncfusion_flutter_pdf :
flutter:3.13.6
syncfusion_flutter_pdf: ^23.1.38

I have the following issue that when i try to read the attached pdf file the pages are blank or mostly unreadable. This is my code for reading a specific page : 

```
Future<Map<String, dynamic>> getPdfContent(String path,{pageIndex=1}) async{


  PdfDocument document = PdfDocument(inputBytes: await _readDocumentData(path));
  int pageCount = document.pages.count;

  if (pageIndex < 0 || pageIndex >= pageCount) {
    print("Error: Page index out of bounds!");
    document.dispose();
    return {'text': '', 'pageCount': pageCount};
  }

  PdfTextExtractor extractor = PdfTextExtractor(document);

  // Since you're extracting from a single page, start and end page indices should be the same
  String text = extractor.extractText(startPageIndex: pageIndex, endPageIndex: pageIndex);

  document.dispose();

  return {'text': text, 'pageCount': pageCount};
}
```

The provided code is working fine with other pdf files, am i doing something wrong and can this issue be resolved somehow?

Thank you for your time!
Stanko

[Черният Лебед - Насим Талеб.pdf](https://github.com/syncfusion/flutter-widgets/files/12784591/-.pdf)

**Comment by spstrademark** [comment]
Would like to ask is there a workaround using this plugin or i need to find another solution?
Thank you!

**Comment by jeyalakshmit** [comment]
We have confirmed the issue ”Extracting text returns incorrect results in specific PDF document” as a defect in our product and fix will be included in our upcoming weekly release, which will be available on February 27, 2024.

**Comment by jeyalakshmit** [comment]
The fix for the reported issue is included in our weekly release v24.2.8.
 
Root cause: Incorrect decoding of embedded content in PDF document causes failure while extracting text.

Kindly use the following link to refer to our latest release version,
https://pub.dev/packages/syncfusion_flutter_pdf/versions

**Comment by spstrademark** [comment]
Hello,

thanks! I can confirm that the issue is resolved.

Best regards,
Stanko
